R/0_documentation.R

```diff
@@ -15,6 +23,22 @@ rd_case_definition <- function(type = "param") {
 }
 
 
+rd_contact_basis <- function(type = "param") {
+  checkmate::assert_choice(type, c("param", "field"))
+  paste("(`list(list())`)\\cr",
+        "A nested list with all the needed information for the contact_basis\\cr",
+        "* `counts` contains the age stratified contact counts across the arenas of the basis",
+        "  (e.g. 'work', 'home', 'school', 'other')\\cr",
+        "* `prop` contains a list of the proportion of population in 5-year age-groups\\cr",
```

Comment:
I do not find that we should specify 5yr groups - it is 'just' the default in what we supply with the package. Suggest: "... population in each age group."